### PR TITLE
Create user

### PR DESCRIPTION
**Pull request by ckvuby**

### CODE REVIEW

File: models/users.js [hunk 1]
```diff
@@ -4,24 +4,47 @@ class Users {
...
+    var reason = null;
+    this.list.forEach((user) => {
+      if (user.fakeName == fakeName) {
+        success = false
+        reason = "fakeName is already taken"
+      }
+      if (user.realName == realName) {
+        success = false
+        reason = "This real name is already taken. Maybe add your last name?"
+      }
```
Comment (TinyGobby) [inline models/users.js]: Maybe make these display on screen instead of a console log?

File: routes.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const Users = require('./models/users');
+
+var users = new Users();
+
+console.log("Here again");
+
```
Comment (TinyGobby) [inline routes.js]: Should this be removed?

File: test/models/users.test.js [hunk 3]
```diff
@@ -18,6 +18,20 @@ describe("Users", () => {
...
+
+    it("should not add a user if fakename is taken", () => {
+      users.add("unicorn42", "seb");
+      expect(users.add("unicorn42", "vu").success).toBe(false);
+    })
+
+    it("should not add a user if realname is taken", () => {
+      users.add("unicorn99", "vu");
+      expect(users.add("unicorn88", "vu").success).toBe(false);
+    })
```
Comment (TinyGobby) [inline test/models/users.test.js]: Nice tests

**Comment by TinyGobby** [comment]
[Connect user sign in on front end with user model on backend](https://trello.com/c/uqdepbxx/69-connect-user-sign-in-on-front-end-with-user-model-on-backend)

File: models/users.js (hunk 1 above)
Comment (IvyMic) [reply models/users.js]: Nice work guys

Comment (sebgrebe) [reply models/users.js]: Yes, will put error messages on display! :)

**Pull request merged**